code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -125,8 +114,23 @@
 			add_objective(hijack_objective)
 			return
 
+	// See if they get the global target.
+	if(objective_count <= objective_amount && length(SSticker.mode.traitors) >= 3 && SSticker.mode.VIP_target != owner)
+		var/assassin_count = length(SSticker.mode.protect_target_assassins)
+		var/protector_count = length(SSticker.mode.protect_target_protectors)
+
+		// Make a new protector if there is no target set yet (first protector) or at a 20% chance if there are less or equal protectors compared to assassins
+		if(!SSticker.mode.VIP_target || (protector_count <= assassin_count && prob(10)))
```

Comment:
There's a mismatch here between the comment and code. Comment says 20% chance, code is 10%.

Reply:
Changed in the last PR. Danke!